### PR TITLE
III-4435 - New Organizer form

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Added

- Added new organizer form

---

Ticket: https://jira.uitdatabank.be/browse/III-4435

### CODE REVIEW

**Review by brampauwelyn — COMMENTED** [review]
@Anahkiasen For me it's ok ✅

File: src/pages/steps/AdditionalInformationStep/PhysicalLocationStep.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { StackProps } from '@/ui/Stack';
+
+type PhysicalLocationStepProps = StackProps & TabContentProps & StepProps;
+
+function PhysicalLocationStep({
+  onValidationChange,
+  ...props
+}: PhysicalLocationStepProps) {
+  const location = props.watch('location');
+  useEffect(() => {
```
Comment (simon-debruijn) [inline src/pages/steps/AdditionalInformationStep/PhysicalLocationStep.tsx:18]:
```suggestion
  const location = props.watch('location');
  
  useEffect(() => {
```

File: src/pages/steps/LocationStep.tsx [hunk 2]
```diff
@@ -1188,6 +1183,10 @@ const locationStepConfiguration: StepsConfiguration<'location'> = {
   validation: yup.lazy((value) => {
     const url = window.location.href;
 
+    if (url.includes('organizers') && url.includes('create')) {
+      return yup.object();
```
Comment (simon-debruijn) [inline src/pages/steps/LocationStep.tsx:1187]: shouldn't the validation be stricter?

File: src/pages/steps/Steps.tsx [hunk 3]
```diff
@@ -195,7 +200,7 @@ const Steps = ({
   };
 
   return (
-    <Stack spacing={5} width="100%">
+    <Stack {...getStackProps(props)} spacing={5} width="100%">
```
Comment (simon-debruijn) [inline src/pages/steps/Steps.tsx:203]: Its more useful to spread the `getStackProps` after existing props, so they can be overridden if needed

**Review by simon-debruijn — APPROVED** [review]

File: src/pages/steps/LocationStep.tsx (hunk 2 above)
Comment (brampauwelyn) [reply src/pages/steps/LocationStep.tsx:1187]: @simon-debruijn in the the case of "organizer creation" the location is not mandatory that's why we return an object here...

**Pull request merged**